tools/cdhit/cd_hit_est.xml

```diff
@@ -20,16 +20,14 @@
   
   <inputs>
     <param name="fasta_in" type="data" format="fasta" label="EST Sequences to cluster"/>
-    <param name="similarity" type="float" value="0.9"  label="similarity threshold: .75 - 1.0, default is .9">
-      <validator type="in_range" message="sequence similarity threshold should be .75 - 1.0" min=".75" max="1.0"/>
+    <param name="similarity" type="float" value="0.9"  label="similarity threshold">
+        <help> similarity threshold is included behind 0.75 and 1.0
```

Comment:
Here, it is best to get `help="Similarity threshold must be in [0.75;1] interval"`